lib/python/qmk/cli/path.py

```diff
@@ -0,0 +1,8 @@
+from milc import cli
```

Comment:
This probably doesnt need to exist, given we can do `qmk env QMK_FIRMWARE`.

Reply:
Done.